File: Computer-Seekho-.NET/Computer_Seekho_DN/Models/Course.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;
using Microsoft.EntityFrameworkCore;

namespace Computer_Seekho_DN.Models;

[Table("course")]
public partial class Course
{
    [Key]
    [Column("course_id")]
    public int CourseId { get; set; }

    [Column("course_descriptor")]
    [StringLength(500)]
    public string? CourseDescriptor { get; set; }

    [Column("course_duration")]
    public int? CourseDuration { get; set; }

    [Column("course_fee")]
    public double? CourseFee { get; set; }

    [Column("course_is_active", TypeName = "bit(1)")]
    public ulong? CourseIsActive { get; set; }

    [Column("course_name")]
    [StringLength(100)]
    public string? CourseName { get; set; }

    [Column("course_syllabus")]
    [StringLength(200)]
    public string? CourseSyllabus { get; set; }

    [Column("cover_photo")]
    [StringLength(100)]
    public string? CoverPhoto { get; set; }

    [InverseProperty("Course")]
    [JsonIgnore]
    public virtual ICollection<Batch> Batches { get; set; } = new List<Batch>();

    [InverseProperty("Course")]
    [JsonIgnore]
    public virtual ICollection<Student> Students { get; set; } = new List<Student>();

    [InverseProperty("Course")]
    [JsonIgnore]
    public virtual ICollection<VideoMaster> VideoMasters { get; set; } = new List<VideoMaster>();
}

File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/Entities/Course.java
package com.Project.Entities;

import jakarta.persistence.*;
import lombok.Data;

@Table(name = "course")
@Entity
@Data
public class Course {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "course_id")
    private int courseId;

    @Column(length = 100, name = "course_name") 
    private String courseName;

    @Column(length = 500, name = "course_descriptor") 
    private String courseDescription;

    @Column(name = "course_duration")
    private int courseDuration;

    @Column(length = 200, name = "course_syllabus") 
    private String courseSyllabus;

    @Column(name = "course_fee")
    private double courseFee;

    @Column(name = "course_is_active")
    private boolean courseIsActive =  true;

    @Column(length = 100, name = "cover_photo") 
    private String coverPhoto;
    
    
}